_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -39,12 +39,15 @@ For each test target:
 For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
```

Comment:
Expectation 1 is currently not generating a set of instrument (even though it should). Also, Expectation 1 can generate 0 instruments in the case of the focus condition.

See https://github.com/act-rules/act-rules.github.io/pull/1276#discussion_r432342251